### DISCUSSION TITLE
EOL of Adobe Air v33.1.1.932b

### DISCUSSION THREAD

**Discussion by aussiewombat** [airsdk/Adobe-Runtime-Support#2513] [General]
Hello there, does anyone know Adobe Air v33 remains a supported version?

**Comment by ajwfrost** [comment]
Hi

It's still available and you're very welcome to continue using it .. we haven't got any plans to update it unless there are critical errors found in it and where people aren't happy with then updating to the 50.x branches. There may be some limitations to this imposed by third parties like Google Android/Apple iOS, so e.g. there may be updates we can't make to the 33.1 branch without having to update the OS requirements which we won't want to do, but I would hope that this doesn't occur too often since we're just looking at bug fixes really..

Likewise, 33.1.1.743/744 are still supported which are the ones with the old WebKit libraries in them (supported except for on Apple Silicon..) and we'll do the same there, updating with bug fixes where it's critical.

thanks

**Reply by OrangeGirlop** [reply]
So Adobe air is still supported by harman? In what way and to what extend? Im not aware of any cves being published.

**Reply by ajwfrost** [reply]
This thread (I believe!) is about the 33.1.1.x branch. But more generally, yes we continue to support AIR, see https://airsdk.harman.com. It's a mixture of some bug fixes (including ones we've introduced, of course!), OS compatibility (which is quite a fair effort for the mobile platforms!), and feature updates (new APIs, trying to provide the features that most people would find useful, plus optimisations etc)

CVEs = critical vulnerabilities? I'm not aware of any either. AIR is a framework/runtime though, so it's basically a case of running an application on a client computer. So to exploit any vulnerability, the bad actor would need to already have access to your computer, or your application would need to go out and find some malicious content and try to execute it (which we could argue means that the vulnerability is in the application, rather than the framework, although it may be possible for us to close the door on any such exploit) -> if anything does get reported here, we will of course provide the appropriate patches/updates!
